### PR TITLE
Feat docs site

### PR DESCRIPTION
**Pull request by raman1999**
Added docs html and styling 

Preview site : https://rocket-ui-library.netlify.app/

### CODE REVIEW

**Review by RahulSingh9131 — APPROVED** [review]
Website is looking great raman, everything in perfectly mentioned. Go ahead and merge.

File: stylesheets/docs.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  height: calc(100vh - 1rem);
+}
+
+/* sidebar list item */
+#main-list > .list-item {
+  padding: 0;
+  margin: 0;
+  text-align: center;
+}
+#main-list > .list-item > a {
```
Comment (rahulcs754) [inline stylesheets/docs.css:38]: don't make high Specificity. please try to make your code with one class and two class only

**Review by rahulcs754 — APPROVED** [review]
i looked at your code please check and think about it

File: stylesheets/docs.css (hunk 1 above)
Comment (raman1999) [reply stylesheets/docs.css:38]: thanks for suggestion, actually I have used it to override  style in my list.css file that's why to make it high specific

**Pull request merged**